fideslog/api/routes/events.py

```diff
@@ -0,0 +1,135 @@
+from logging import getLogger
+
+from fastapi import APIRouter, Depends, HTTPException, Request, status
+from sqlalchemy.exc import DBAPIError
+from sqlalchemy.orm import Session
+
+from fideslog.api.config import config
+from fideslog.api.database.data_access import create_event
+from fideslog.api.database.database import get_db
+from fideslog.api.schemas.analytics_event import AnalyticsEvent
```

Comment:
Changing these to relative imports is working just fine for me:
```suggestion
from ..config import config
from ..database.data_access import create_event
from ..database.database import get_db
from ..schemas.analytics_event import AnalyticsEvent
```